### ISSUE TITLE
Samsung devices

### ISSUE THREAD

**Issue by eremeev23** [naver/egjs-view360#427]
## Description
Black window instead of panorama image (https://ibb.co/9Hshmdx). This issue is on samsung smartphones only (in all browsers).
Errors in console -- https://ibb.co/Sf24Km6
## Project
Nuxt3 + "@egjs/vue3-view360": "^4.0.0-beta.4"

**Comment by eremeev23** [comment]
Nvmr. Fixed by reducing size of image from 4096x2048 for 2048x1024

**Comment by nguyenph194812** [comment]
i have the same issue , but for me it seems that all smartphones get black window instead of panorama image. Do we have any fix for this ?

**Comment by malangfox** [comment]
@nguyenph194812 
Does the issue you commented on occur with all panorama images?
If not, can we check the panorama image that reproduces your issue?

**Comment by nguyenph194812** [comment]
@malangfox yes, i do resize my image from 8192*4092 to 2048*1024 and it shows the image, but the original one is black. The bug reproduces on all smartphones. Here's my image
https://drive.google.com/file/d/1PA8rMR1bskrTIWHqRQv5JUMXoPQqGAiO/view?usp=drive_link

**Comment by nguyenph194812** [comment]
Also i wonder does your lib use Multiresolution like krpano, marzipano ?

**Comment by nguyenph194812** [comment]
@malangfox hei, does the bug fixed yet. i use android and it always shows black, get the same log as @eremeev23

**Comment by rawthriver** [comment]
+1
